internal/resources/fabric/service_token/resource_test.go

```diff
@@ -63,7 +63,54 @@ func TestAccFabricServiceToken_PNFV(t *testing.T) {
 	})
 }
 
-func testAccFabricServiceTokenConfig(serviceTokenName string, serviceTokenDescription string, virtualDeviceUuid string) string {
+func TestAccFabricAsidePortServiceToken_PNFV(t *testing.T) {
```

Comment:
This tests needs to be with PPDS suffix if we're going to use the ppds port on line 70. Is PPDS used for this test?

Reply:
We are going to use pnfv user, I will update this line:
portUuid = ports["ppds"]["dot1q"][0].GetUuid()